api/api.go

```diff
@@ -325,5 +327,23 @@ func GetRepoContributors(url string) []Contributor {
 		}
 	}
 
-	return contributors
+	// Need to get top three contributions
+	var c1, c2, c3 int = 0, 0, 0
+	var tot int = 0
+
+	for _, c := range contributors {
+		tot += c.RecentCommits
+		if c.RecentCommits > c1 {
+			c3 = c2
+			c2 = c1
+			c1 = c.RecentCommits
+		} else if c.RecentCommits > c2 {
+			c3 = c2
+			c2 = c.RecentCommits
+		} else if c.RecentCommits > c3 {
+			c3 = c.RecentCommits
+		}
+	}
+
+	return c1 + c2 + c3, tot, nil
```

Comment:
is the difference between c1+c2+c3 = contributions only from top 3
and tot = all contributions gone through?

Reply:
Yes